frontend/src/v4/modules/reducers.ts

```diff
@@ -68,6 +69,7 @@ export default function createReducer(history) {
 		currentUser: currentUserReducer,
 		userManagement: userManagementReducer,
 		dialog: dialogReducer,
+		dialogs: dialogsReducer,
```

Comment:
`dialog` and `dialogs` is going to be confusing as hell.

Can we name this as`dialogsV5` or something?